version.inc

```diff
@@ -11,4 +11,4 @@ if(NOT DEFINED BBL_INTERNAL_TESTING)
 set(BBL_INTERNAL_TESTING "1")
 endif()
 set(SLIC3R_VERSION "01.03.01.01")
-set(SoftFever_VERSION "1.3.4")
+set(SoftFever_VERSION "1.3.4 PA")
```

Comment:
We usually don't change version info for individual fixes.
The version info will be updated in the next release with other changes.

-SoftFever